octobot_trading/personal_data/orders/order.py

```diff
@@ -315,8 +328,11 @@ async def default_exchange_update_order_status(self):
     def generate_executed_time(self):
         return self.exchange_manager.exchange.get_exchange_current_time()
 
+    def is_counted_in_available_funds(self):
```

Comment:
I think we'll need to discuss about it, I had the same issue in futures development. I think we need to do more than that